lib/teams/aws-batch/aws-batch-on-eks-team.ts

```diff
@@ -0,0 +1,167 @@
+import { assertEC2NodeGroup } from "../../cluster-providers";
+import { ClusterInfo, Values } from "../../spi";
+import { ApplicationTeam, TeamProps } from "../team";
+import { KubectlProvider, ManifestDeployment } from "../../addons/helm-addon/kubectl-provider";
+import { loadYaml, readYamlDocument } from "../../utils/yaml-utils";
+import { Construct } from "constructs";
+import { createNamespace } from "../../utils";
+
+import * as batch from 'aws-cdk-lib/aws-batch';
+import * as eks from 'aws-cdk-lib/aws-eks';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import { AutoScalingGroup } from "aws-cdk-lib/aws-autoscaling";
+import * as iam from "aws-cdk-lib/aws-iam";
+
+const NAMESPACE = 'aws-batch';
+
+/**
+ * Interface to define an AWS Batch on EKS team
+ */
+export interface BatchEksTeamProps extends TeamProps {
+  /**
+   * Name of the team
+   */
+  name: string,
```

Comment:
why name and namespace repeated here? they are inherited from the TeamProps, let's drop them.

Reply:
addressed